esp/esp/tagdict/__init__.py

```diff
@@ -445,6 +446,14 @@
         'category': 'manage',
         'is_setting': True,
     },
+    'grade_increment_date': {
+        'is_boolean': False,
+        'help_text': 'When should students\' grades rollover/increment?',
+        'default': datetime.date(datetime.date.today().year, 7, 31),
+        'category': 'manage',
```

Comment:
PR works great, but I think this tag should be in the "student registration settings" portion of the global tags since that's where the other grade-related tags are, and this only affects students.